server/src/test/java/io/crate/integrationtests/SQLTypeMappingTest.java

```diff
@@ -189,27 +182,27 @@ public void testInvalidInsertIntoObject() throws Exception {
             }))
             .hasPGError(INTERNAL_ERROR)
             .hasHTTPError(BAD_REQUEST, 4000)
-            .hasMessageContaining("Cannot cast");
+            .hasMessageContaining("Cannot cast object element `created` with value `true` to type `timestamp with time zone`");
```

Comment:
This change is enough to cover the case, without the fix, you get: `cannot cast .... to type 
'object'`